apps/web/src/index.css

```
@@ -214,6 +214,7 @@
   --animate-menu-open: menu-open 120ms cubic-bezier(0.16, 1, 0.3, 1);
   --animate-typing-dot: typing-dot 1s infinite;
   --animate-swipe-out: swipeOut 100ms ease-out;
+  --animate-swipe-out-left: swipeOutLeft 100ms ease-out;
```

Stylelint (17.14.0)

[error] Expected "swipeOutLeft" to be "swipeoutleft" (value-keyword-case)

(value-keyword-case)